### PR TITLE
Feat/accordion

### PR DESCRIPTION
**Pull request by MarioIpedis**
-accordion component
-test
-readme

### CODE REVIEW

File: packages/accordion/src/components/accordion/accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+      this.accPanels = this.el.shadowRoot
+        .querySelector('#ip-accordion')
+        .querySelectorAll('.js-panel');
+
+      this.setSlotId();
+
+      this.hidePanels();
+
+      this.openFirstPanel();
+    }, 0);
```
Comment (RomaricMitiana) [inline packages/accordion/src/components/accordion/accordion.tsx:70]: ca you explain the reason of the setTimeOut?

File: packages/accordion/src/components/accordion/accordion.tsx [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+
+  // For accessibility - so as not to make elements inside tabulable
+  // - After timeout, so as to make element inside rendered first to be able to determine height
+  hidePanels() {
+    setTimeout(() => {
+      const panels = Array.from(this.accPanels).filter(
+        (_panel: HTMLElement, index) => {
+          return index != 0;
+        },
+      );
```
Comment (RomaricMitiana) [inline packages/accordion/src/components/accordion/accordion.tsx:175]: shift() method on the array should give the same result

**Review by RomaricMitiana — CHANGES_REQUESTED** [review]
We should find a way to avoid setTimeout() usage.

File: packages/tab-panel/src/README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@
 Install the ip-tab-panel component as a dependency in the project:
 
 ```bash
-npm install ip-radio
+npm install ip-tab-panel
 ```
```
Comment (patrickIpedis) [inline packages/tab-panel/src/README.md:11]: This is confusing so next time better to use a dedicated branch

**Pull request merged**